cmd/kots/cli/set-config.go

```diff
@@ -228,3 +234,19 @@ func getConfigValuesFromArgs(v *viper.Viper, args []string) ([]byte, error) {
 
 	return b, nil
 }
+
+func logConfigValidationErrors(groupValidationErrors []configtypes.ConfigGroupValidationError, log *logger.CLILogger) {
+	log.Errorf("Following config items have validation errors:")
+	for _, groupValidationError := range groupValidationErrors {
+		log.Errorf("Group - %s(%s)", groupValidationError.Title, groupValidationError.Name)
+		log.Errorf("  Items:")
+		for _, itemValidationError := range groupValidationError.ItemErrors {
+			log.Errorf("    Name: %s", itemValidationError.Name)
```

Comment:
Let's use `title` instead of `name`.

Reply:
`kot set config`/`ConfigValues` use `name` to update values.
Isn't using name be more appropriate?

Reply:
hmm, yeah maybe. ok